CONTRIBUTING.md

```diff
@@ -163,6 +163,22 @@ Notebooks now have really efficient ways to share your code and display your out
 - [Binder](https://mybinder.org/) that lets you turn a Git repository into a collection of interactive notebooks!
 - [Voila](https://github.com/voila-dashboards/voila) is a newcomer that will allow to build dashboads for your presentations!
 
+#### 2.6 Release a new build 
+
+To release a new version on Pypi, go at the root of the repository, and trigger the build with `pip`.
```

Comment:
Not sure what "trigger the build with `pip`" means: should the user run a `pip` command he is supposed to know of already?